src/encoded/tests/test_batch_download.py

```diff
@@ -328,7 +328,7 @@ def test_batch_download_report_download(testapp, workbook):
         b'Post-synchronization time', b'Post-synchronization time units',
         b'Replicates',
     ]
-    assert len(lines) == 64
+    assert len(lines) == 65
```

Comment:
This increased by 1 because you added a new experiments. Now, what is special about the one you added? The data looks like the others.

Reply:
Because I didn’t want to use another experiment to test specifically one with index reads files, in case that removed a test of something else.